.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+/templates                              @jobcloud/backend-lead-dev
```

Comment:
This CODEOWNERS file is probably copied from another place, this project doesn't have those directories